scripts/envDFL.sh

```diff
@@ -485,6 +485,26 @@ launch_sa() {
     unset LD_PRELOAD
 }
 
+launch_nsa() {
+    if [ ! -f "$2" ]; then
+        error_exit "IIDM network file $2 doesn't exist"
+    fi
+    if [ ! -f "$3" ]; then
+        error_exit "DFL configuration file $3 doesn't exist"
+    fi
+    if [ ! -f "$4" ]; then
+        error_exit "Security Analysis contingencies file $4 doesn't exist"
+    fi
+    export_preload
+    "$MPIRUN_PATH" -np $NBPROCS $DYNAFLOW_LAUNCHER_INSTALL_DIR/bin/DynaFlowLauncher \
+    --log-level $DYNAFLOW_LAUNCHER_LOG_LEVEL \
+    --network $2 \
+    --config $3 \
+    --contingencies $4 \
+    --flat # Steady state calculations.
```

Comment:
I am not sure to like the name "flat", as it will introduce confusion with flat/warm options

Reply:
the choice is yours!

Reply:
--nsa ?